share/plugins/notenames-interactive.qml

```diff
@@ -64,9 +64,11 @@ MuseScore {
     function getChordName(chord) {
         var text = "";
         var notes = chord.notes;
+        var sep = "\n";   // change to "," if you want them horizontally (anybody?)
         for (var i = 0; i < notes.length; i++) {
-            var sep = "\n";   // change to "," if you want them horizontally (anybody?)
-            if ( i > 0 )
+            if ((curScore.selection.elements.length && !notes[i].selected) || !notes[i].visible)
+                continue // skip notes a that are not selected or invisible
+            if (text.text) // only if text isn't empty
```

Comment:
Just `if (text)` here

Reply:
OK, this here works too though

Reply:
Ah, only for this plugin, right, thanks for catching!